src/functions/template-tags/html.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * HTML functions (template-tags) for use in WordPress templates.
+ */
+use Tribe\Utils\Element_Classes;
+
+/**
+ * Parse input values into a valid array of classes to be used in the templates.
+ *
+ * @since  TBD
+ *
+ * @param  mixed $classes,... unlimited Any amount of params to be rendered as classes.
+ *
+ * @return array
+ */
+function tribe_get_classes() {
+	$element_classes = new Element_Classes( func_get_args() );
+	return $element_classes->get_classes();
+}
+
+/**
+ * Parses input values into a valid class html attribute to be used in the templates.
+ *
+ * @since  TBD
+ *
+ * @param  mixed $classes,... unlimited Any amount of params to be rendered as classes.
+ *
+ * @return string
+ */
+function tribe_classes() {
+	$element_classes = new Element_Classes( func_get_args() );
+	echo $element_classes->get_attribute();
```

Comment:
I think adding `esc_attr` to escape the attribute value would be great here as well.

Reply:
Used `esc_html()` since `esc_attr` would remove the correct `"'` usage around the `class=""`

Reply:
Oh I see I think instead of writing the ` class="" ` might be better to return only the result that goes inside passing the controller to the caller to use something like: `class="<?php tribe_classes(); ?>"`.

Kind of the same way https://github.com/JedWatson/classnames works.